### ISSUE TITLE
Compatibility fix - Fixing autoplay in Cordova Android apps

### ISSUE THREAD

**Issue by Techn1c4l** [mebjas/html5-qrcode#198]
I've been dealing with this problem for a very long time.

The issue is the video stream does not start automatically when being implemented in a Cordova Android app. Eventually I've ran out of solutions and just resorted to simulating a screen tap which indeed helped.

But today I've found the true reason of this behavior.

**Setting `muted` property of an HTMLVideoElement will not count as muted video for a browser**.

**This does not work**:

`videoElement.muted = true;`

You have to add the `muted=true` attribute.

**This is the proper way**:

`videoElement.setAttribute('muted', true);`

Basically as I understand, the browser will ignore `autoplay` attribute if it doesn't detect a `muted` attribute.

Please implement this fix. There is a lot of frustrated people out there who were not able to solve this for years.

**Comment by mebjas** [comment]
Thanks for the bug and suggestion, merged to master. Will be published soon!
